src/test/kotlin/org/objectionary/deog/unit/graph/TestBase.kt

```diff
@@ -29,6 +29,8 @@ import kotlin.test.assertEquals
 
 /**
  * Common interface for all test classes
+ *
+ * @todo #29:90m/DEV Integration tests needs to be added
```

Comment:
@c71n93 Looks like this puzzle description too short.

Reply:
@Graur Done. Please check.